met-web/src/App.tsx

```diff
@@ -95,9 +97,13 @@ const App = () => {
     const getTranslationFile = async () => {
         try {
             const translationFile = await import(`./locales/${language}/${tenant.id}.json`);
+            const title = translationFile?.header?.title || document.title;
```

Comment:
that's not how we get values from the translationFile. Check     const { t: translate } = useAppTranslation(); in the landingComponent for example